src/primitives/transaction.h

```diff
@@ -1056,9 +1056,9 @@ struct CMutableTransactionBase
     int32_t nVersion;
     std::vector<CTxIn> vin;
 
-protected:
-    std::vector<CTxOut> vout;
+// protected:
 public:
+    std::vector<CTxOut> vout;
```

Comment:
Please restore `vout` protected accessibility and use public functions `addOut` and `getOut` within test `test_createnewblock.cpp`.